src/package.ps1

```diff
@@ -187,6 +187,9 @@ function InstallPackage { # $db, $status
 		if ($db.metadatadb.$old.refcount -gt 0) {
 			$db.metadatadb.$old.refcount -= 1
 		}
+		if ($db.metadatadb.$old.refcount -eq 0) {
```

Comment:
```suggestion
		} else if ($db.metadatadb.$old.refcount -eq 0) {
```

Reply:
This needs to be just `if`, not `else if`.